MobilityData/mobility-feed-api
# feat: add recaptcha to signup screen

**Summary:**

 Add Google reCaptcha to the Sign-up screen.

Out of scope:
 - reCaptcha component on the CompleteRegistration. This will be done as part of a different PR once #164 is merged.
 
Part of #152 

**Expected behavior:** 

The sign-up screen contains a reCaptcha and is required for submitting a sign-up request. 

<img width="909" alt="Screenshot 2023-11-15 at 4 41 40 PM" src="https://github.com/MobilityData/mobility-feed-api/assets/1192523/c50ca2f7-2e70-4229-a995-ba6a790309f3">

**Testing tips:**


- Go to the sign-up screen
- Verify that reCaptcha is on the screen
- Fill in all the required information without clicking the reCaptcha checkbox
- Verify that the reCaptcha error message appears on the screen, _"You must verify you are not a robot."_
- Click the reCaptcha checkbox and click the sign-up button
- Verify that the sign-up request is sent to the screen, and if it is successful, the user is redirected to the accounts page



Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Format the title like "feat: [new feature short description]." The title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
